Blog/spec/models/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it "requires title to be present" do
+      n = Post.new()
+
+      n.valid?
+
+      expect(n.errors.messages).to have_key(:title)
+    end
+
+    it "requires body to be present and be at least 50chars long" do
+      n = Post.new(body: "baaa")
```

Comment:
There is a better way to do these validation checks: https://github.com/thoughtbot/shoulda-matchers

I am not sure this gem is used in class, but I make use of it.